src/registrar/abha-components/biometric-authentication/biometric-authentication.component.ts

```diff
@@ -70,13 +70,14 @@ export class BiometricAuthenticationComponent implements OnInit {
           if (captureres != null) {
             // Assign captureres to capturePID
             this.rddeviceService.capturePID = captureres;
-            this.matDialogRef.close(captureres);
-            if (this.aadharBioNum != null && this.aadharBioNum != undefined) {
-              this.showProgressBar = true;
-              this.generateAbha(); // Call generateAbha after setting capturePID
-            } else if (this.healthDataNum != null && this.healthDataNum != undefined) {
-              this.rddeviceService.capturePID = captureres;
-            }
+            console.log("rdservice captures:", this.rddeviceService.capturePID)
+            this.matDialogRef.close(this.rddeviceService.capturePID);
+            // if (this.aadharBioNum != null && this.aadharBioNum != undefined) {
```

Comment:
Pls remove this commented line of codes

Reply:
need some changes in this will be pushed in next commit